src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/OreCrusher.java

```diff
@@ -50,6 +50,12 @@ protected void registerDefaultRecipes(List<ItemStack> recipes) {
 
         recipes.add(new ItemStack(Material.MAGMA_BLOCK, 4));
         recipes.add(SlimefunItems.SULFATE);
+
+        recipes.add(SlimefunItems.CARBON);
+        recipes.add(new ItemStack(Material.COAL, 8));
+
+        recipes.add(SlimefunItems.COMPRESSED_CARBON);
+        recipes.add(new SlimefunItemStack(SlimefunItems.CARBON, 4));
```

Comment:
This can just be an ItemStack

Reply:
It cannot. There is no such constructor for ItemStacks.